st2actions/st2actions/query/mistral/v2.py

```diff
@@ -42,13 +42,25 @@ def query(self, execution_id, query_context):
             status, output = self._get_workflow_result(exec_id)
             if output and 'tasks' in output:
                 LOG.warn('Key conflict with tasks in the workflow output.')
+        except requests.exceptions.ConnectionError:
+            msg = 'Unable to connect to mistral.'
```

Comment:
That's true, but it could make it easier for user to debug and spot issues due to invalid configuration (e.g. maybe mistral is down or mistral url specified in the config is invalid).